dev-tools/mage/gotest.go

```diff
@@ -93,6 +95,32 @@ func makeGoTestArgsForModule(name, module string) GoTestArgs {
 	return params
 }
 
+// fetchGoPackages uses "go list ./module/..." to get all Go packages for a
+// module.
+// It returns a list of packages that are relative to the module path.
+// Example: for module "kafka" it'll return:
```

Comment:
Suggestion to clarify how this interacts with directory structure: `for module "kafka"` -> `for module "kafka" inside "metricbeat/module"`